src/rastervision/samples/workflow-configs/segmentation/deeplab-test-remote.json

```diff
@@ -0,0 +1,100 @@
+{
+    "train_scenes": [
+        {
+            "id": "2-10",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_10_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_10_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "src_classes": [ 6, 1 ],
+                    "dst_classes": [ 1, 0 ]
```

Comment:
The class_ids should start at 1.

Reply:
I'm guessing these are indices into the channel dimension. It would be good to clarify that.

Reply:
These are not indices, please see the (rewritten) response above.  All of these will be well documented.